### ISSUE TITLE
Edit-FalconScript - Relative path incorrectly assumes C:\Windows\System32 instead of current location

### ISSUE THREAD

**Issue by datorr2** [CrowdStrike/psfalcon#45]
```powershell
PS C:\Users\datorr2\crowdstrike-scripts> Edit-FalconScript -Id $script.id -Path .\My_Script.ps1 -Name 'My_Script.ps1' -Comment 'Updated to v2.0'
Exception: C:\Program Files\WindowsPowerShell\Modules\PSFalcon\Private\Private.ps1:932
Line |
 932 |  …             $FileStream = [System.IO.FileStream]::New($Formdata.$Key, …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling ".ctor" with "2" argument(s): "Could not find file 'C:\WINDOWS\system32\My_Script.ps1'."
```

**Comment by bk-cs** [comment]
Thank you! I've confirmed this issue, which I missed in testing. 

Please use absolute paths for now when referring to files and I will work to figure out a way to correct this for v2.0.8.

**Comment by bk-cs** [comment]
This issue has been fixed in v2.0.8 which has been pushed, but I have not completed the release itself as I wait on feedback for Issue #48. You can manually install this version by downloading the master repository through GitHub, or wait until the PowerShell Gallery is updated with v2.0.8 after outstanding issues are fixed.
